roles/osasinfra/templates/clientpod.yml.j2

```diff
@@ -0,0 +1,103 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ cifmw_osasinfra_client_pod_name }}
+  namespace: openstack
+spec:
+  containers:
+  - args:
+    - infinity
+    command:
+    - /bin/sleep
+    env:
+    - name: CONFIG_HASH
+      value: n6h58h584h689h588hfdh687h8dh64dh685h5c5h5dfh578h648h5c6h576h54fh559h65bh5bh674h59bh678h655h5ddh55hb7h7hb4h5bch69h668q
+    - name: OS_CLOUD
```

Comment:
I would remove this. It's creating a mess in the pod (by default is using the admin project and sometimes it's misleading the user and the developer)

Reply:
Done